PgpWordList.Test/PgpWordAggregationParserTest.cs

```diff
@@ -0,0 +1,18 @@
+using Xunit;
+
+namespace Messerli.PgpWordList.Test
+{
+    public class PgpWordAggregationParserTest
+    {
+        [Theory]
+        [ClassData(typeof(PgpWordListTestData))]
+        public void ParsePgpWordAccumulation(byte[] expectedPgpBytes, string separator, string pgpString)
```

Comment:
Make this a verb

Reply:
Addressed in https://github.com/messerli-informatik-ag/pgp-word-list/pull/1/commits/ee7e0ee16c845190597975a16154bd59ab465523